### DISCUSSION TITLE
Clarification: Can I Customize the In-App Messaging with a pop up with two buttons?

### DISCUSSION THREAD

**Discussion by nihp** [firebase/firebase-ios-sdk#11441] [Ideas]
I need to show a prompt using the In-App Messaging. Can I show that using the In App Messaging popups. Here I need to show a popup with would you like to update the app or not with two buttons say cancel and update in iOS swift app.

Is this will be possible in the In-App Messaging.

**Comment by nihp** [comment]
Can I give the action buttons as Yes and No for Primary and Secondary actions?

@ncooke3

**Reply by ncooke3** [reply]
Hi @nihp, I'm cc'ing @eldhosembabu who may be able to provide better guidance here!

**Comment by eldhosembabu** [comment]
You should be able to create a card message with update and cancel button from firebase console like here:
<img width="701" alt="Screenshot 2023-06-21 at 10 13 59 AM" src="https://github.com/firebase/firebase-ios-sdk/assets/5833071/5ee195f5-9add-45cd-b42f-a79dbf52dfc5">
